lib/runner.go

```diff
@@ -75,29 +119,31 @@ func (r *Runner) getTestScriptsWithOrder() (string, []string, error) {
 		return "", nil, err
 	}
 	sort.Strings(testFiles)
-	if !r.InOrder {
+	if !r.options.inOrder {
 		r.shuffleOrder(testFiles)
 	}
 	return testRoot, testFiles, err
 }
 
 func (r *Runner) getTestScripts() (string, []string, error) {
-	includeRe, err := regexp.Compile(r.IncludeRe)
+	includeRe, err := regexp.Compile(r.options.includeReStr)
 	if err != nil {
 		return "", nil, fmt.Errorf("Error parsing files to include: %s", err)
 	}
-	excludeRe, err := regexp.Compile(r.ExcludeRe)
+	excludeRe, err := regexp.Compile(r.options.excludeReStr)
 	if err != nil {
 		return "", nil, fmt.Errorf("Error parsing files to exclude: %s", err)
 	}
 
 	var foundTests []string
-	for _, testFolderOriginal := range r.TestTargets {
-		testFolder, err := filepath.Abs(testFolderOriginal)
+	for _, testFolderOriginal := range r.options.testTargets {
+		var testFolder string
```

Comment:
I should just be able to get rid of this var. e.g `testFolder, err := filepath.Abs(testFolderOriginal)`

Reply:
You are right. I was getting a shadowing linting issue with another code (this code is changing so much).

Reply:
`vetshadow` is pointing the finger at me for shadowing `err` on line 129. I'll ignore it.